### PR TITLE
feat: IND

### PR DESCRIPTION
**Pull request by ArnavD74**
Location is IND
Able to scrape, but currently can not collect static population data from local csv

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@ dist/
 node_modules
 .DS_Store
 .vscode
+coronadatascraper-cache/*
+coronadatascraper-cache
```
Comment (qgolsteyn) [inline .gitignore]: This should be left out, `coronadatascraper-cache` is the submodule we use to store the cache for this project.

**Comment by qgolsteyn** [comment]
Alright, population works! But I still notice that the crawlere doesn't recognize that. Let me investigate.

File: .gitignore (hunk 1 above)
Comment (ArnavD74) [reply .gitignore]: My bad, I had not meant to include it initially.

**Comment by lazd** [comment]
@ArnavD74 great work! Wondering why Jharkhand is not on the map?

image

**Comment by ArnavD74** [comment]
> @ArnavD74 great work! Wondering why Jharkhand is not on the map?
> 
> image

It seems to be on the map for me currently? Can you check again and report

**Pull request merged**